domains/finhane.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "finhane",
```

must be your gh username
```suggestion
    "username": "felipefinhane",
```